### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by LucaGabi** [Dotnet-Boxed/Templates#375] [Q&A]
﻿**Which project template is the issue about?**
API
...

Hi,
Dose this support jwt and refresh tokens ?
Fluent model validation ?

**Comment by VictorioBerra** [comment]
They do not come with the templates but those are trivial to add after you generate a template.

For example just generate a new api and then add the FluentValidation.AspNetCore and register it in your startup and you're done.

**Comment by RehanSaeed** [comment]
Thanks @VictorioBerra for answering.

**Comment by LucaGabi** [comment]
Hi what about the first question on jwt api auth ?

**Comment by RehanSaeed** [comment]
It supports everything that ASP.NET Core supports. So yes it does but you need to read the docs and add that code yourself.
